src/views/DashboardView/DashboardView.js

```diff
@@ -123,9 +111,4 @@ const mapStateToProps = (state) => ({
   numWifi: countWifi(state),
   numWifiDwellers: countWifiDwellers(state)
 });
-export default connect((mapStateToProps), {
-  eventsRequest,
-  readersRequest,
-  tagsRequest,
-  usersRequest
-})(DashboardView);
+export default connect((mapStateToProps), {})(DashboardView);
```

Comment:
Much of this is copied straight from: https://github.com/davezuko/react-redux-starter-kit/blob/v3.0.0-alpha.0/src/routes/Counter/containers/CounterContainer.js

However, even there they don't have the extra parenthesis, at least not anymore.